lib/options.ts

```diff
@@ -22,7 +22,8 @@ var knownOpts: any = {
 		"client": String,
 		"available": Boolean,
 		"release": Boolean,
-		"debug": Boolean
+		"debug": Boolean,
+		"validValue": Boolean
```

Comment:
no, we have two options here: add valid-value as known option or change validValue to be only valid.
Ping @Fatme @ikoevska  - which one do you think is better?